zoo-css-challenge/style.css

```
@@ -85,13 +89,14 @@ footer {
   border-radius: 50%;
   display: inline-block;
   position: fixed;
-  bottom: 20px;
+  top: 20px;
   right: 20px;
 
   display: flex;
   justify-content: center;
   align-content: center;
   flex-direction: column;
+  transition: ease 2s;
```

```suggestion
  transition: background-color 2s ease;
```
The documentation says we should provide the properties in this order

https://developer.mozilla.org/en-US/docs/Web/CSS/transition